mp/mpfshell.py

```diff
@@ -633,6 +634,12 @@ def do_repl(self, args):
                 self.fe.cache = {}
 
             self.fe.setup()
+            try:
+                self.fe.cd(pwd)
+            except IOError:
+                pass
```

Comment:
Can you add a comment here explaining when this exception might be thrown?

Reply:
Not sure; comments do not seem to fit the local style. It would be the second comment in a 800+ lines file. Is it worth it?

Reply:
Well, I just added a comment :)

Reply:
If the style is to leave no comments, the style is wrong :P